packages/wouter/src/index.js

```diff
@@ -319,11 +321,16 @@ export const Redirect = (props) => {
   const { to, href = to } = props;
   const [, navigate] = useLocation();
   const redirect = useEvent(() => navigate(to || href, props));
+  const { ssrContext } = useRouter();
 
   // redirect is guaranteed to be stable since it is returned from useEvent
   useIsomorphicLayoutEffect(() => {
     redirect();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  if (ssrContext) {
+    ssrContext.redirectTo = to;
+  }
```

Comment:
We already use optional chaining in the library, so it is safe to write:

```suggestion
  ssrContext?.redirectTo = to;
```

Reply:
This fails linting: "Parsing error: Optional chaining cannot appear in left-hand side".

Reply:
My bad, optional chaining doesn't work with assignments.